src/lender/assessor/base.sol

```diff
@@ -185,6 +199,7 @@ contract BaseAssessor is Math, Auth {
         if (tranche == junior || minJuniorRatio == 0) {
             return true;
         }
+        _drip();
```

Comment:
supplyApprove should update the debt as well.

Reply:
same here, having this function as a view would be really helpful

Reply:
*note